lib/config/config.go

```diff
@@ -130,8 +130,10 @@ type Configuration struct {
 	Devices        []DeviceConfiguration `xml:"device" json:"devices"`
 	GUI            GUIConfiguration      `xml:"gui" json:"gui"`
 	Options        OptionsConfiguration  `xml:"options" json:"options"`
-	IgnoredDevices []protocol.DeviceID   `xml:"ignoredDevice" json:"ignoredDevices"`
-	IgnoredFolders []string              `xml:"ignoredFolder" json:"ignoredFolders"`
+	IgnoredDevices []ObservedDevice      `xml:"remoteIgnoredDevice" json:"remoteIgnoredDevices"`
+	IgnoredFolders []ObservedFolder      `xml:"remoteIgnoredFolder" json:"remoteIgnoredFolders"`
```

Comment:
It's a new thing prefixed with "remote" in json, the old thing is just gone.

Reply:
And it will just get ignored, and I am ok with that. I don't see any other sensible way to handle this. We can give everyone that is using this a heads up I guess.

Reply:
@imsodin is probably right about the config breaking the wrappers. For syncthing-android the config model and restApi functions ignoredevice, ignorefolder, unignoredeviceandfolders would have to be updated.
It would be great if temporarily the old and the new config ignore parts would be valid to help getting a smooth transition.

Reply:
Get will simply not return anything, post will just ignore things, so yeah, things might not work, but then on the other hand we're in control of Android release cycle so I can make sure these things get dealt with.